### PR TITLE
[Refactoring] Moving pkg/v1/tkg as a top level module

### PR DESCRIPTION
**Pull request by sathyanarays**
### What this PR does / why we need it
Moving pkg/v1/tkg as a top level go module
### Which issue(s) this PR fixes

Fixes #

### Describe testing done for PR

Tested by running the github pipelines

### Release note

```release-note

```

### CODE REVIEW

File: tkg/kappclient/client.go [hunk 1]
```diff
@@ -23,8 +23,8 @@ import (
 	kapppkg "github.com/vmware-tanzu/carvel-kapp-controller/pkg/apiserver/apis/datapackaging/v1alpha1"
 	secretgenctrl "github.com/vmware-tanzu/carvel-secretgen-controller/pkg/apis/secretgen2/v1alpha1"
 
-	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/constants"
-	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/tkgpackagedatamodel"
+	"github.com/vmware-tanzu/tanzu-framework/tkg/constants"
+	"github.com/vmware-tanzu/tanzu-framework/tkg/tkgpackagedatamodel"
```
Comment (maralavi) [inline tkg/kappclient/client.go:27]: Is it possible that this subdirectory remain where it is? I'm working to moving it to its own subdirectory and this change will cause big merge conflict

File: tkg/test/tkgpackageclient/package_plugin_integration_test.go [hunk 2]
```diff
@@ -21,19 +21,19 @@ import (
 	"k8s.io/apimachinery/pkg/util/wait"
 	"k8s.io/client-go/util/retry"
 
-	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/test/framework/exec"
+	"github.com/vmware-tanzu/tanzu-framework/tkg/test/framework/exec"
```
Comment (maralavi) [inline tkg/test/tkgpackageclient/package_plugin_integration_test.go:24]: Is it possible that this subdirectory ```tkg/test/tkgpackageclient``` remain where it is? I'm working to moving it to its own subdirectory and this change will cause big merge conflict

File: tkg/tkgpackageclient/client.go [hunk 3]
```diff
@@ -4,7 +4,7 @@
 package tkgpackageclient
 
 import (
-	"github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/kappclient"
+	"github.com/vmware-tanzu/tanzu-framework/tkg/kappclient"
```
Comment (maralavi) [inline tkg/tkgpackageclient/client.go:7]: Is it possible that the subdirectories ```tkgpackageclient```, ```tkgpackagedatamodel```,```kappclient``` and ```tkg/test/tkgpackageclient``` remain where it is? I'm working to moving those to their own subdirectory and this change will cause big merge conflict

**Comment by Vandy-P** [comment]
reopening Pr for testing Cayman , Bolt build .

**Comment by Vandy-P** [comment]
Closing the PR , we will test it with Latest PRs

**Pull request closed without merging**